php/class-customize-snapshot-manager.php

```diff
@@ -647,16 +267,19 @@ public function enqueue_controls_scripts() {
 
 		wp_enqueue_style( 'customize-snapshots' );
 		wp_enqueue_script( 'customize-snapshots' );
+
```

Comment:
Might as well add `$post = null;` here so that static analysis (in PhpStorm) doesn't complain about a potentially-undefined variable `$post` later.